### PR TITLE
Update search-the-audit-log-in-security-and-compliance.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about a searching audit log via another method as we can use PS to search the audit log - Reference https://docs.microsoft.com/en-us/microsoft-365/compliance/audit-log-search-script?view=o365-worldwide

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8699

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 1]
```diff
@@ -1125,7 +1125,13 @@ Yes. The Office 365 Management Activity API is used to fetch the audit logs prog
 
 **Are there other ways to get auditing logs other than using the security and compliance portal or the Office 365 Management Activity API?**
 
-No. These are the only two ways to get data from the auditing service.
+Yes, You can retrieve audit logs by using the following methods:
+
+- [The Office 365 Management Activity API](/office/office-365-management-api/office-365-management-activity-api-reference)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
- The [Office 365 Management Activity API](/office/office-365-management-api/office-365-management-activity-api-reference).
```

File: microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md [hunk 2]
```diff
@@ -1125,7 +1125,13 @@ Yes. The Office 365 Management Activity API is used to fetch the audit logs prog
...
+
+- The [audit log search tool](search-the-audit-log-in-security-and-compliance.md) in the Microsoft Purview compliance portal
+
+- The [Search-UnifiedAuditLog](/powershell/module/exchange/search-unifiedauditlog) cmdlet in Exchange Online PowerShell
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/search-the-audit-log-in-security-and-compliance.md]:
```suggestion
- The [audit log search tool](search-the-audit-log-in-security-and-compliance.md) in the Microsoft Purview compliance portal.

- The [Search-UnifiedAuditLog](/powershell/module/exchange/search-unifiedauditlog) cmdlet in Exchange Online PowerShell.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**